File: domain/src/main/java/no/nav/melosys/domain/dokument/arbeidsforhold/Fartsomraade.java
package no.nav.melosys.domain.dokument.arbeidsforhold;

import no.nav.melosys.domain.kodeverk.Kodeverk;

public enum Fartsomraade implements Kodeverk {


    INNENRIKS("innenriks", "Innenriks"),

    UTENRIKS("utenriks", "Utenriks");

    private String kode;
    private String beskrivelse;

    Fartsomraade(String kode, String beskrivelse) {
        this.kode = kode;
        this.beskrivelse = beskrivelse;
    }

    @Override
    public String getKode() {
        return kode;
    }

    @Override
    public String getBeskrivelse() {
        return beskrivelse;
    }
}

File: domain/src/main/kotlin/no/nav/melosys/domain/dokument/arbeidsforhold/Fartsomraade.kt
package no.nav.melosys.domain.dokument.arbeidsforhold

import no.nav.melosys.domain.kodeverk.Kodeverk


enum class Fartsomraade(private val kode: String, private val beskrivelse: String) : Kodeverk {
    INNENRIKS("innenriks", "Innenriks"),
    UTENRIKS("utenriks", "Utenriks");

    override fun getKode(): String = kode

    override fun getBeskrivelse(): String = beskrivelse
}
